squad-create-reproducer

```diff
@@ -97,9 +114,13 @@ def parse_args(raw_args):
 
 def run(raw_args=None):
     args = parse_args(raw_args)
+    if not args.custom_command and not args.local:
```

Comment:
if not custom_command and --local is set then we get the the tuxrun reproducer also...

Reply:
Removed as this warning is no longer needed now we changed the code to support fetching of TuxTest reproducers in this case.